apps/wallet-dashboard/components/Dialogs/SendToken/SendTokenDialog.tsx

```diff
@@ -51,19 +54,29 @@ function SendTokenDialogBody({
         isPayAllIota,
     );
 
-    function handleTransfer() {
+    async function handleTransfer() {
```

Comment:
Instead of writing 2 new useState s for both error and loading, you could just turn it into a `useMutation`